MediaBrowser.Model/System/SystemStorageInfo.cs

```diff
@@ -0,0 +1,62 @@
+#nullable disable
+#pragma warning disable CS1591
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using MediaBrowser.Model.Updates;
+
+namespace MediaBrowser.Model.System;
+
+/// <summary>
+/// Contains informations about the systems storage.
+/// </summary>
+public class SystemStorageInfo
```

Comment:
I would suggest adding a separate DTO class in case we ever add stuff here that should NOT be in API responses.